chord_metadata_service/restapi/schema_utils.py

```diff
@@ -148,3 +148,50 @@ def schema_list(schema):
         "type": "array",
         "items": schema
     }
+
+
+def patch_project_schemas(base_schema: Dict, extension_schemas: Dict[str, object]) -> Dict:
```

Comment:
`: dict` and `-> dict` instead is preferable if you don't need properties typing